### ISSUE TITLE
Test failure inTransformationsPluginsTest

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#14906]
When running the testsuite locally, I get this error:

```
PhpMyAdmin\Tests\Plugins\Transformations\TransformationPluginsTest::testTransformation with data set #25 (PhpMyAdmin\Plugins\Transformations\Text_Plain_Longtoipv4 Object (), array(4294967295.0, array('option1', 'option2')), '255.255.255.255')
long2ip() expects parameter 1 to be int, float given
```

When run on Travis, it does not seem to have the error.

This happens with both QA_4_8 and master.

**Comment by mauriciofauth** [comment]
I could not reproduce that. What is the value of your `PHP_INT_MAX` constant?

**Comment by ibennetch** [comment]
2147483647

**Comment by mauriciofauth** [comment]
So it's a 32 bit system?

When a value is greater than `PHP_INT_MAX`, it is converted to float automatically.

**Comment by ibennetch** [comment]
It is a 32-bit system, yes.

**Comment by williamdes** [comment]
Reproduced using https://hub.docker.com/r/i386/ubuntu/ on my 64 bit system

**Comment by williamdes** [comment]
Fixed by #14906
